### ISSUE TITLE
Is macro still working ?

### ISSUE THREAD

**Issue by rrey-tot** [justmiles/go-markdown2confluence#31]
Hi there,

Using:
```
$ markdown2confluence --version
markdown2confluence 3.1.2
```

I tried the example snipets for ToC from the README:
```markdown
   ```CONFLUENCE-MACRO
    name:toc
      minLevel:2
    ```
```

It is not rendered in Confluence cloud.
Is it still working or did I miss something ?

**Comment by justmiles** [comment]
You know what I don't believe I've cut a release since that feature was added. I'll cut one this morning.

**Comment by justmiles** [comment]
These changes are available in release [v3.2.0](https://github.com/justmiles/go-markdown2confluence/releases/tag/v3.2.0).

**Comment by rrey-tot** [comment]
thanks !
